scripts/convert_gen2_to_gen3_hsc.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# This script will convert the hsc repo from gen2 to gen3.  This conversion requires the
+# raws, so compress_jointcal_hsc_raw_test_data.py must have been run prior to this script.
+# Until the gen3 repo format is fully stable, only these conversion scripts will be
+# available in the rpo, and not the generated gen3 repo.  After the gen3 format is
+# stable enough, we can commit the output gen3 repo.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Run the gen2 to gen3 conversion code
+butler convert hsc -i lsst.obs.subaru.HyperSuprimeCam --gen2root `realpath hsc`
```

Comment:
I've fixed that bug so you can use `./hsc` now.

Reply:
I guess I'm working off the weekly, but I can change this.